scripts/yaml-generator/src/config.lisp

```diff
@@ -370,7 +370,7 @@ common ones to include"
    :name "Backends/llvm"
    :resolver 17.3
    :path-to-other "../../"
-   :packages (list *standard-library* *core* *context* *pipeline* *frontend* *sexp* *translate*)
+   :packages (list *standard-library* *core* *context* *pipeline* *frontend* *sexp* *translate* *datastructures*)
```

Comment:
Same here?

Reply:
Needed to run `stack test` on the llvm package